src/apps/relay/mainrelay.c

```diff
@@ -1604,7 +1604,7 @@ static void read_config_file(int argc, char **argv, int pass)
 		if (full_path_to_config_file)
 			f = fopen(full_path_to_config_file, "r");
 
-		if (f && full_path_to_config_file) {
+		if (f) {
```

Comment:
I am not understand why you removed here the full_path_to_config_file. Can you help?

Reply:
I removed it because the code inside the if does not use the `full_path_to_config_file` variable. It only depends on the opened file handler `f`. But I can put it back if you thinks that's better.